webpack/scenes/Subscriptions/__tests__/SubscriptionValidations.test.js

```diff
@@ -51,21 +51,25 @@ describe('recordsValid', () => {
     expect(recordsValid([])).toBe(true);
   });
 
+  /* eslint-disable object-curly-newline */
   it('accepts valid array', () => {
     const rows = [
-      { quantity: 10, available: 10, availableQuantity: 100 },
+      { quantity: 10, available: 10, availableQuantity: 100, upstream_pool_id: ' ' },
       { quantity: 10, available: 10, availableQuantity: -1 },
       { quantity: -1, available: -1 },
-      { quantity: 10, available: 10 },
+      { quantity: 10, available: 10, upstream_pool_id: ' ' },
     ];
     expect(recordsValid(rows)).toBe(true);
   });
+  /* eslint-enable object-curly-newline */
 
   it('detects invalid record', () => {
+    /* eslint-disable object-curly-newline */
```

Comment:
What changed to require disabling the linter for object-curly-newline? I don't think I see how it applies

Reply:
I'm also unsure about what changed, but eslint definitely fails without this disable/enable.